internal_ws/.cargo/config.toml

```diff
@@ -0,0 +1,10 @@
+# cargo will walk up the filesystem looking for other `.cargo/config.toml`s.
+# Without overrides here, it will find the external workspace's config and turn
+# off optimisation for release builds. We just restore the default values for
```

Comment:
The first two sentences make sense, I think, but I don't understand the third one fully. Maybe the whole comment can be tweaked to something like: "cargo walks up the filesystem looking for `.cargo/config.toml`s which means that it finds the external workspace's config which sets `opt-level` to <whatever value?>. In order to stop that happening, we have to force the `opt-level` we want here." ?